walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -12,12 +12,103 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Figure out what everything is 
+// Find events for attendies, to show conflicts
+// Sort by start time
+// Search for conflicts
+// TODO: Find times where attendies can meet without conflict (Time ranges for mettings without conflict)
+
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
+import java.util.List;
 
 public final class FindMeetingQuery {
+  // @param Event (Collection) - All the events existing on an organization calendar: name, time range, attendies...
```

Comment:
These look like they're meant to be javadoc, but they don't quite conform to the javadoc syntax. I'd either make them proper javadoc or remove them